forge-gui/res/cardsfolder/upcoming/psionic_ritual.txt

```diff
@@ -0,0 +1,11 @@
+Name:Psionic Ritual
+ManaCost:4 U U
+Types:Sorcery
+K:Replicate:tapXType<1/Horror>
+A:SP$ ChangeZone | Origin$ Graveyard | Destination$ Exile | TgtPrompt$ Choose target instant or sorcery card in your graveyard | ValidTgts$ Instant.YouCtrl,Sorcery.YouCtrl | RememberChanged$ True | SubAbility$ DBCopy | SpellDescription$ Exile target instant or sorcery card from a graveyard and copy it. You may cast the copy without paying its mana cost.
+SVar:DBCopy:DB$ Play | Defined$ Remembered | Amount$ All | CopyCard$ True | WithoutManaCost$ True | SubAbility$ ExileSelf
+SVar:ExileSelf:DB$ ChangeZone | Origin$ Stack | Destination$ Exile | Defined$ Self | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+DeckNeeds:Type$Horror
+AI:RemoveDeck:All
```

Comment:
Set up appropriate conditions for the AI and see if it plays this card in a semi-logical way.

If it does, then you don't need this tag.

Reply:
So the AI didn't play this card at an appropriate time when conditions were good?

Reply:
Judging by the looks of the card itself, the AI should manage to play things like that, but might be worth testing again? Maybe something broke at some point or I'm missing something about the card that the AI doesn't handle well :/ At any rate, judging by the card specification alone (Oracle text etc.), I don't see anything that should prevent the AI from playing it.

Reply:
Tried this again with several Horrors in play, plenty of instants and sorceries in the yard, more than enough mana, and at a distinct disadvantage on the board, with 1 life left. I don't think conditions could get much worse, and the AI didn't play it. Some of the cards I based this on had this tag, some did not. It's up to you guys, but at least let the card in for now until time permits for AI improvements.

Reply:
Alright, I'll try to take a look at the AI part later :)